frontend/src/App.js

```diff
@@ -297,7 +302,15 @@ class App extends Component {
   axiosGraphQL
     .post('', { query: CREATE_ITEM} )
     .then(
-      results => {this.setState({results: results.data.data.createItem.items})},
+      results => {
+        if (results.data.errors && results.data.errors.length > 0){
```

Comment:
This simply adds better error-handling. If GraphQL returns a response but that response contains errors, they will be displayed to the user.